src/index.ts

```diff
@@ -41,10 +41,18 @@ const backportPullRequest = async ({
   repo: RepoName;
   title?: PullRequestTitle;
 }): Promise<PullRequestNumber> => {
+  const {
+    data: { title: originalTitle },
+  } = await octokit.pulls.get({
+    number: pullRequestNumber,
+    owner,
+    repo,
+  });
+
   const {
     body = `Backport #${pullRequestNumber}.`,
     head = `backport-${pullRequestNumber}-on-${base}`,
-    title = `Backport #${pullRequestNumber} on ${base}`,
+    title = `Backport #${pullRequestNumber} on ${base}: ${originalTitle}`,
```

Comment:
Thanks a lot for this fix, live is a lot easier now!!

Could this be reformulated to "Backport (${base}): ${originalTitle}" instead? That would leave more room for the original title and I believe the pull request number is much more useful inside the main comment on the new PR where it is clickable.